### PR TITLE
add separated spec for micro OS

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: package/yast2-schema-micro.spec [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+Group:	        System/YaST
+License:        GPL-2.0-or-later
+
+Url:            https://github.com/yast/yast-schema
+
+# provide schema, so it is full replacement
+Provides:       yast2-schema
+# both cannot be used as it uses same files
+Conflicts       yast2-schema
```
Comment (imobachgs) [inline package/yast2-schema-micro.spec:34]: The colon symbol is missing here.

File: package/yast2-schema-micro.spec [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+BuildRequires: yast2-registration >= 4.3.12
+%endif
+# Added lsm 'none' section to security.rnc
+BuildRequires: yast2-security >= 4.4.7
+BuildRequires: yast2-services-manager
+# tag home_btrfs_subvolume
+BuildRequires: yast2-users >= 4.1.11
+
+
+#!BuildIgnore: yast2-build-test
```
Comment (imobachgs) [inline package/yast2-schema-micro.spec:70]: Could not get rid of all these comments?

**Review by imobachgs — COMMENTED** [review]
In general, it looks good, although it is a pity that we cannot install multiple schema packages at the same time.

File: package/yast2-schema-micro.spec (hunk 1 above)
Comment (jreidinger) [reply package/yast2-schema-micro.spec:34]: ah, good catch, I found it during testing and forget to submit.

File: package/yast2-schema-micro.spec (hunk 2 above)
Comment (jreidinger) [reply package/yast2-schema-micro.spec:70]: well, it is not comments, I think it is directives for build service to ignore some dependencies...I just worry that some of them are already obsolete

**Review by imobachgs — APPROVED** [review]

**Pull request merged**